packages/dynamite/dynamite/README.md

```diff
@@ -2,6 +2,35 @@ Provides a [Dart Build System](https://github.com/dart-lang/build) builder for g
 
 The builder generates code if it find files with an `.openapi.json` or `.openapi.yaml` extension in the lib directory.
 
+# Setup
+
+The generated code makes use of various third party packages. Please add the following dependencies to your `pubspec.yaml`.
+```yaml
+dependencies:
+    - built_collection
+    - built_value
+    - collection
+    - dynamite_runtime
+    - meta
+    - universal_io
+    - uri
+dev_dependencies:
+    - build_runner
+    - built_value_generator
+    - dynamite
```

Comment:
This syntax is wrong since these are maps and not lists.

When we figure out the constraints in https://github.com/nextcloud/neon/pull/1228 we can already add them here.

Reply:
I've used the latest major versions for now and I'll use the same versions in #1228